resources/images: Intermittent failure of TestMetaInvalidSource

```text
--- FAIL: TestMetaInvalidSource (0.00s)
    meta_integration_test.go:203: 
        error:
          no substring match found
        container:
          `failed to create config from result: failed to decode "imaging": invalid metadata source "foo" in imaging.meta.sources config; must be one of [xmp exif iptc]`
        want:
          `invalid metadata source "foo" in imaging.meta.sources config; must be one of [exif iptc xmp]`
        stack:
          D:/a/hugo/hugo/resources/images/meta/meta_integration_test.go:203
            b.Assert(err.Error(), qt.Contains, `invalid metadata source "foo" in imaging.meta.sources config; must be one of [exif iptc xmp]`)
        
FAIL
```

--- a/resources/images/config.go
+++ b/resources/images/config.go
@@ -496,7 +496,9 @@ func (cfg *ImagingConfig) init() error {
 		for i, s := range cfg.Meta.Sources {
 			cfg.Meta.Sources[i] = strings.ToLower(s)
 			if !validMetaSources[cfg.Meta.Sources[i]] {
-				return fmt.Errorf("invalid metadata source %q in imaging.meta.sources config; must be one of %s", s, slices.Collect(maps.Keys(validMetaSources)))
+				keys := slices.Collect(maps.Keys(validMetaSources))
+				slices.Sort(keys)
+				return fmt.Errorf("invalid metadata source %q in imaging.meta.sources config; must be one of %s", s, keys)
 			}
 		}
 	}

